cmd/user-list/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+)
+
+var testCases = []struct {
```

Comment:
Since this isn't shared, treat it like we do all our other test cases and declare it in the function it is used.